charts/apiclarity/templates/fuzzer-template-configmap.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "apiclarity.name" . }}-fuzzer-template
+  namespace: '{{ .Release.Namespace }}'
+  labels:
+    {{ include "apiclarity.labels" . }}
+data:
+  config: |-
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      namespace: '{{ .Release.Namespace }}'
+      labels:
+{{- toYaml (index .Values "apiclarity" "labels") | nindent 8 }}
+    spec:
+      backoffLimit: 0
+      ttlSecondsAfterFinished: 300
+      template:
+        metadata:
+          labels:
+{{- toYaml (index .Values "apiclarity" "labels") | nindent 12 }}
+        spec:
+          restartPolicy: Never
+          volumes:
+          - name: tmp-volume
+            emptyDir: {}
+          securityContext:
+            capabilities:
+              drop:
+              - all
+            runAsNonRoot: true
+            runAsGroup: 1001
+            runAsUser: 1001
+            privileged: false
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+          containers:
+          - name: fuzzer
+            image: "{{ .Values.apiclarity.modules.fuzzer.docker.image }}"
+            imagePullPolicy: Always
+            volumeMounts:
+              - mountPath: /tmp
+                name: tmp-volume
+            env:
+            - name: PLATFORM_TYPE
```

Comment:
what about `FUZZER_PLATFORM_HOST_FROM_FUZZER`?

Reply:
I Did what we discussed in team webex